src/components/Checkbox/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+export const checkboxDataTestIds = {
+  base: 'checkbox__base',
+};
+
+interface CheckboxProps {
+  isValueChecked: boolean;
+}
+
+const Checkbox = ({ isValueChecked }: CheckboxProps): JSX.Element => {
+  return (
+    <div className="flex text-white" data-test-id={checkboxDataTestIds.base}>
+      <input
+        type="checkbox"
+        id="checkbox"
+        checked={isValueChecked}
+        onChange={() => {
+          // Do nothing
+        }}
+        className="peer relative shrink-0 appearance-none w-[28px] h-[30px] border-[1px] border-white rounded-full mt-1 checked:bg-white"
+      />
+
+      {/* FIXME: Cannot apply the class attribute to svg file so I add the svg tag manually */}
+      <svg
```

Comment:
There are a [few ways](https://blog.logrocket.com/how-to-use-svgs-react/) to use SVG in React.
If we don't use any libraries for SVGs, could you make a new component for this SVG?

Reply:
@hoangmirs ah yes, we can do it. Let me update it 👍

Reply:
@hoangmirs updated in [9cf9cc7](https://github.com/manh-t/react-survey/pull/79/commits/9cf9cc7e64b19e65e275327d6b0981dc4379edc3)